ethereum/spec/src/main/java/tech/pegasys/teku/spec/schemas/SchemaDefinitionsAltair.java

```diff
@@ -41,6 +42,7 @@ public class SchemaDefinitionsAltair extends AbstractSchemaDefinitions {
   private final ContributionAndProofSchema contributionAndProofSchema;
   private final SignedContributionAndProofSchema signedContributionAndProofSchema;
   private final MetadataMessageSchemaAltair metadataMessageSchema;
+  private final LightClientBootstrapSchema lightClientBootstrapSchema;
```

Comment:
Added only for Altair schema definitions. Question whether the schema should be added for all consensus versions (i.e. lift up into `SchemaDefinitions`) (though there is no change from Altair) or whether it should only be specified in `SchemaDefinitionsAltair` and used this way.

Reply:
https://github.com/ethereum/beacon-APIs/blob/master/apis/beacon/light_client/bootstrap.yaml#L31

Reply:
We'll want to pull it into `SchemaDefinitions` I think, rather than hard references to Altair, as that creates the altair schema definitions even though maybe only this one is used... @ajsutton may have opinions here so tagging

Reply:
Given the light client types are defined in the spec they will be versioned with hard forks and so need to come from `SchemaDefinitions`.  They will still just wind up being added to `SchemaDefinitionsAltair` because `SchemaDefinitionsBellatrix` extends from that and doesn't need to make any changes so will just inherit the Altair definitions.

Since these types are new in Altair I think you've actually got it right that they get added to `SchemaDefinitionsAltair` - they're then inherited by Bellatrix and they don't need to be in Phase0.

Reply:
Not entirely sure that comment made sense... `SchemaDefinitions` is an interface that only contains the types that are common to all hard forks.  For things new in Altair like this you'd call `SchemaDefinitions.toVersionAltair()` to get a `SchemaDefinitionsAltair` and then can use these new methods.

And Bellatrix inherits all the Altair types, then just redefines a few, so `SchemaDefinitionsBellatrix` extends `SchemaDefinitionsAltair` so these methods will also be available there (and Capella extends Bellatrix).

Reply:
Sorry should have responded to this earlier, but it looks like you worked it out and done the right thing. The new method only gets added to `SchemaDefinitionsAltair`.  We can be a bit sloppy with terminology when saying things should be added to "SchemaDefinitions" - we mean to the schema definitions for the first milestone it's relevant to.